src/app/view/manager.rs

```diff
@@ -741,42 +872,22 @@ impl ManagerSignView {
             ))));
         }
         col = col.push(card::white(Container::new(signer)));
-        Container::new(scroll(
-            &mut self.scroll,
-            Container::new(
-                Column::new()
-                    .push(
-                        Row::new()
-                            .push(
-                                Column::new()
-                                    .push(
-                                        button::transparent(
-                                            &mut self.back_button,
-                                            Container::new(text::simple("Go Back")).padding(10),
-                                        )
-                                        .on_press(Message::Previous),
-                                    )
-                                    .width(Length::Fill),
-                            )
-                            .push(
-                                Container::new(
-                                    button::cancel(
-                                        &mut self.cancel_button,
-                                        Container::new(text::simple("X Close")).padding(10),
-                                    )
-                                    .on_press(Message::Menu(Menu::Home)),
-                                )
-                                .width(Length::Shrink),
-                            ),
-                    )
-                    .push(
-                        Container::new(col)
-                            .width(Length::Fill)
-                            .align_x(Align::Center),
-                    )
-                    .spacing(20),
-            ),
-        ))
+        Container::new(
+            Column::new()
+                .push(header)
+                .push(
+                    Container::new(text::bold(text::simple("Sign transaction")))
+                        .width(Length::Fill)
+                        .align_x(Align::Center),
+                )
+                .push(scroll(
+                    &mut self.scroll,
+                    Container::new(col)
+                        .width(Length::Fill)
+                        .align_x(Align::Center),
+                ))
```

Comment:
Sadly the column inside is not centered
![2021-05-28T18:01:23,803733728+02:00](https://user-images.githubusercontent.com/6933020/120011993-3273e380-bfdf-11eb-800d-1e92db1f46a9.png)

Reply:
Please check if it's resolved :)

Reply:
fixed for me